transcom/mymove
# INT-B-19095

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A916575&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Added a button for creating moves as a services counselor

### How to test

1. Enable isCounselorMoveCreateEnabled FF
2. Login as a Services Counselor
3. Search
4. Create Move Button should be on the page and navigates to edit user info page
5. Login as a QAE
6. Search
7. Create Move button does not exist 
8. Profit
